src/services/__tests__/anchor-service.test.ts

```diff
@@ -715,6 +720,70 @@ describe('anchor service', () => {
       const updatedRequest = await requestRepository.findByCid(toCID(request.cid))
       expect(updatedRequest.status).toEqual(RequestStatus.COMPLETED)
     })
+
+    test('Request succeeds without anchor for already CIDs with next CIDs that have been anchored', async () => {
```

Comment:
grammar of this test description seems off, I don't really understand what it's trying to say?

Reply:
i'll change this another PR because integration tests take too long. Fuzzy brain did this word dump haha. What i really mean to say is: 

Request succeeds without anchor if subsequent CIDs are already anchored